### ISSUE TITLE
Any example of samehead or sametail would help

### ISSUE THREAD

**Issue by dlamblin** [xflr6/graphviz#204]
I don't quite see how edges can get samehead or sametail applied.

**Comment by xflr6** [comment]
Hi, caveat that this might be better be directed at upstream https://graphviz.org/documentation/ ([repo](https://gitlab.com/graphviz/graphviz/)).

- https://www.graphviz.org/docs/attrs/samehead/
- https://www.graphviz.org/docs/attrs/sametail/

Does the example in https://stackoverflow.com/a/5442607 maybe help?

**Comment by dlamblin** [comment]
Thanks I did find adding it in the attributes to edge worked, but only after I stopped using the ortho curvestyle. It seems upstream has this bug: https://gitlab.com/graphviz/graphviz/-/issues/1856 it affects samehead, sametail, and headport and tailport too.
One thing I noticed was that lhead and ltail for clipping to the outside of a cluster was not working in the produced dot output from here because while there's _head and _head_static there's no _head_compound or something to apply compound=true with.
